Formula/xplanet.rb

```diff
@@ -1,8 +1,8 @@
 class Xplanet < Formula
   desc "Create HQ wallpapers of planet Earth"
   homepage "http://xplanet.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/xplanet/xplanet/1.3.0/xplanet-1.3.0.tar.gz"
-  sha256 "44fb742bb93e5661ea8b11ccabcc12896693e051f3dd5083c9227224c416b442"
+  url "https://downloads.sourceforge.net/project/xplanet/xplanet/1.3.1/xplanet-1.3.1.tar.gz"
+  sha256 "4380d570a8bf27b81fb629c97a636c1673407f4ac4989ce931720078a90aece7"
   revision 2
```

Comment:
(This should have been pointed out by the `brew audit --strict --online <formula>`.)